src/main/kotlin/org/malachite/estella/services/HrPartnerService.kt

```diff
@@ -28,7 +27,7 @@ class HrPartnerService(
     }
 
     fun updateHrPartner(id: Int, hrPartner: HrPartner) {
-        val currPartner: HrPartner = hrPartnerRepository.findById(id).get()
+        val currPartner: HrPartner = hrPartnerRepository.findByUserId(id).get()
```

Comment:
Czemu nie korzystasz z getHrPartner?

Reply:
Znaczy ja tu w sumie tylko nazwę metody zmieniłem przez shift+f6, a refactor hrów to przyszły task XD